XLS-20/README.md

```diff
@@ -0,0 +1,828 @@
+|---------------------  |:------------------:|:---------------:|:--------------:|
+| `Flags`               | :heavy_check_mark: | `number`        |    `UINT32`    |
+
+A set of flags that specifies options or controls the behavior of the transaction. This proposal only defines one flag at this time:
+
+> | Flag Name     | Flag Value  | Description                                                                    |
+>|:--------------:|:-----------:|:-------------------------------------------------------------------------------|
+>| `tfSellToken`  | `0x00000001`| If set, indicates that the offer is a sell offer. Otherwise, it is a buy offer. ||
+
+Note that the `Flags` field includes both transaction-specific and generic flags. This proposal only specifies the transaction-specific flags; all currently valid generic flags (e.g., `tfFullyCanonicalSig`) are applicable, but not listed here.
+
+The transactor **SHOULD NOT** allow unknown flags to be set.
+
+---
+
+| Field Name            | Required?        |  JSON Type   | Internal Type   |
+|---------------------  |:----------------:|:------------:|:---------------:|
+| `NFTokenID`            |:heavy_check_mark:|`string`      |    `Hash256`    |
+
+Identifies the **`NFTokenID`** of the **`NFToken`** object that the offer references.
+
+---
+
+| Field Name            | Required?        |  JSON Type        | Internal Type        |
+|---------------------  |:----------------:|:---------------:   |:-----------------:    |
+| `Amount`       |  :heavy_check_mark:      | `Currency Amount`    |  `AMOUNT`         |
+
+Indicates the amount expected or offered for the **`Token`**.
+
+The amount must be non-zero, except where this is an offer is an offer to sell and the asset is XRP; then it is legal to specify an amount of zero, which means that the current owner of the token is giving it away, gratis, either to anyone at all, or to the account identified by the `Destination` field.
+
+---
+
+| Field Name            | Required?        |  JSON Type        | Internal Type        |
+|---------------------  |:----------------:|:---------------:   |:-----------------:    |
+| `Expiration`       |                  |  `number`   |  `UINT32`         |
+
+Indicates the time after which the offer will no longer be valid. The value is the number of seconds since the [Ripple Epoch](https://xrpl.org/basic-data-types.html#specifying-time).
+
+---
+
+| Field Name        | Required?  |  JSON Type  | Internal Type    |
+|-------------------|:----------:|:-----------:|:----------------:|
+| `Destination`     |            | `string`    | `AccountID`      |
+
+Only valid if the `tfSellToken` flag is set.  If present, indicates that this offer may only be accepted by the specified account (either a broker or a buyer). Attempts by other accounts to accept this offer **MUST** fail.
+
+If successful, **`NFTokenCreateOffer`** transaction results in the creation of **`NFTokenOffer`** object.
```

Comment:
If successful, the **`NFTokenCreateOffer`** transaction results in the creation of an **`NFTokenOffer`** object.